recipes/angsd/build.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+binaries="\
+misc/msToGlf \
+misc/contamination \
+misc/contamination2 \
+misc/splitgl \
+misc/thetaStat \
+misc/printIcounts \
+misc/NGSadmix \
+misc/realSFS \
+misc/smartCount \
+misc/supersim \
+angsd \
+"
+BINDIR=$PREFIX/bin
+mkdir -p $BINDIR
+
+make LDFLAGS="-L=${PREFIX}/lib" C_INCLUDE_PATH="-I=${PREFIX}/include" CPLUS_INCLUDE_PATH="-I=${PREFIX}/include"
```

Comment:
I can rewrite it for consistency. In any case, the tests needs to be restarted - it just seems to have been cancelled.